lib/crowbar/init/application.rb

```diff
@@ -31,6 +31,10 @@ class Application < Sinatra::Base
 
       before do
         logger.level = Logger::DEBUG
+        # parse JSON in POST requests
+        if request.request_method == "POST" && request.content_type.start_with?("application/json")
```

Comment:
could you change this to `request.content_type.include?("json")` ? i guess that would be better here, as there could be a custom json type too, that would not match here